proto/config/fw.proto

```diff
@@ -10,6 +10,46 @@ option java_package = "org.lfedge.eve.config";
 
 message ACEMatch {
   // FIXME: We should convert this to enum
```

Comment:
Feel free to remove this FIXME. I don't think it makes sense to replace the strings with enums any more.

Reply:
I would actually prefer enum, but zedcloud would have to keep setting the `string` for older EVE versions anyway, so it is not really worth making the change.
FIXME comment removed.